File: lista 15questoes/15 questoes em python/att06.py
#6. Composição Implemente uma classe Motor e associe-a a uma classe Carro. A classe Carro deve ter um objeto do tipo Motor como um de seus atributos. 
class Motor:
    def __init__(self, potencia):
        self.potencia = potencia

class Carro:
    def __init__(self, marca, modelo, motor):
        self.marca = marca
        self.modelo = modelo
        self.motor = motor

motor1 = Motor(400)
carro1 = Carro("Mitsubishi", "Lancer", motor1)

File: lista 15questoes/15 questoes em java/att06.java
class Motor {
    private int potencia;

    public Motor(int potencia) {
        this.potencia = potencia;
    }
}

class Carro {
    private String marca;
    private String modelo;
    private Motor motor;

    public Carro(String marca, String modelo, Motor motor) {
        this.marca = marca;
        this.modelo = modelo;
        this.motor = motor;
    }
}

Motor motor1 = new Motor(400);
Carro carro1 = new Carro("Mitsubishi", "Lancer", motor1);
